src/Tribe/Editor/Blocks/Tickets.php

```diff
@@ -175,9 +175,11 @@ public function ticket_availability( $tickets = array() ) {
 				continue;
 			}
 
-			$available = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
+			$available     = $ticket->available();
+			$max_at_a_time = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
 
-			$response['tickets'][ $ticket_id ]['available'] = $available;
+			$response['tickets'][ $ticket_id ]['available']    = $available;
+			$response['tickets'][ $ticket_id ]['max_purchase'] = $max_at_a_time;
```

Comment:
@sc0ttkclark would this be impacted by the code you were adding tests to?

Reply:
No, we don't have availability endpoint tests yet